azkaban-spi/src/main/java/azkaban/spi/ExecutorType.java

```diff
@@ -0,0 +1,7 @@
+package azkaban.spi;
+
+public enum ExecutorType {
```

Comment:
Can you please add documentation for this enum?

Reply:
Added documentation.